app/src/test/java/io/apicurio/registry/AbstractRegistryTestBase.java

```diff
@@ -41,9 +42,12 @@ protected String generateArtifactId() {
      * @param resourceName the resource name
      */
     protected final String resourceToString(String resourceName) {
-        InputStream stream = getClass().getResourceAsStream(resourceName);
-        Assertions.assertNotNull(stream, "Resource not found: " + resourceName);
-        return new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"));
+        try (InputStream stream = getClass().getResourceAsStream(resourceName)) {
+            Assertions.assertNotNull(stream, "Resource not found: " + resourceName);
+            return new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"));
+        } catch (IOException e) {
+            return null;
```

Comment:
Null? Why not throw some RuntimeException ... ?

Reply:
OK, I will change it (after the call), NPE is much more fun ;)

Reply:
PR sent